src/_P001_Switch.ino

```diff
@@ -567,22 +597,77 @@ boolean Plugin_001(byte function, struct EventStruct *event, String& string)
               if (loglevelActiveFor(LOG_LEVEL_INFO)) {
                 String log = F("SW  : GPIO=");
                 log += CONFIG_PIN1;
-                log += F(" State=");
+              		log += F(" -> Input=");
                 log += state ? '1' : '0';
-                log += output_value == 3 ? F(" Doubleclick=") : F(" Output value=");
+              		log += eventString;
                 log += output_value;
                 addLog(LOG_LEVEL_INFO, log);
               }
               sendData(event);
 
               // reset Userdata so it displays the correct state value in the web page
-              UserVar[event->BaseVarIndex] = sendState ? 1 : 0;
+//              UserVar[event->BaseVarIndex] = sendState ? 1 : 0;
```

Comment:
why did you disabled this?

Reply:
With the flashing detection the state of the UserVar[event->BaseVarIndex] can be
0, 1 or 2 (flashing)!
A few lines above the commented line you can find the code
UserVar[event->BaseVarIndex] = output_value;

The commented line can be deleted, it was just a reminder for me.